alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -404,19 +405,27 @@ public static function getReplacingEntry($recordedEntry, $asset)
 				$replacingEntry = entryPeer::retrieveByPKNoFilter($replacingEntryId);
 				if ($replacingEntry)
 				{
+					$recordedEntrySegmentCount = $replacingEntry->getRecordedEntrySegmentCounnd(); 
+					if($recordedEntrySegmentCount && $recordedEntrySegmentCount > $liveSegmentCount)
```

Comment:
Better adding default value to getRecordedEntrySegmentCounnd reducing the need to validate recordedEntrySegmentCount